packages/components/tree/src/tree.type.ts

```diff
@@ -63,6 +63,7 @@ export declare interface TreeOptionProps {
   label: string
   disabled: string
   isLeaf?: boolean
+  class?: (data: TreeData, node: Node) => string | { [key: string]: boolean }
```

Comment:
Looks like you didn't include the type of `string` here.

```suggestion
  class?: (data: TreeData, node: Node) => string | { [key: string]: boolean } | string
```